src/middlewared/middlewared/plugins/system.py

```diff
@@ -534,6 +534,7 @@ def debug_download(self, job):
 
 
 class SystemGeneralService(ConfigService):
+    HTTPS_PROTOCOLS = ['TLSv1', 'TLSv1.1', 'TLSv1.2']
```

Comment:
According to nginx TLSv1.3 is already a possibility, why are we not allowing it?